_episodes/02-navigation.md

```diff
@@ -164,11 +166,15 @@ your OS.
 > * **Network filesystem** - Your home directory is an example of a network filesystem. Data stored
 >   here is available throughout the HPC system and files stored here are backed up. Files stored
 >   here are typically slower to access, the data is actually stored on another computer and is
->   being transmitted and made available over the network! `/project` and `/home` are both network
->   filesystems.
+>   being transmitted and made available over the network!
 > * **Scratch** - Some systems may offer "scratch" space. Scratch space is typically faster to use
 >   than your home directory or network filesystem, but is not usually backed up, and should not be
->   used for long term storage. In the last example, `/scratch` is a scratch drive.
+>   used for long term storage.
+> * **Work file system** - As an alternative to (or sometimes as well as) Scratch space, some HPC 
+>   systems offer fast parallel file system access as a work file system. Typically, this will have 
```

Comment:
while it is correct, is it essential to indicate that we are talking about a **parallel** file system here?